docs/Release_notes.md

```diff
@@ -1,7 +1,16 @@
 JPO GeoJSON Converter Release Notes
 ----------------------------
 
-Version 2.0.0
+Version 2.2.0
+----------------------------------------
+
+### **Summary**
+This release includes compatibility upgrades for the jpo-ode 4.1.0 version.
+
+Enhancements in this release:
+- [CDOT PR 13](https://github.com/CDOT-CV/jpo-geojsonconverter/pull/13): Update gjc to work with latest version of the jpo-ode 4.1.0
```

Comment:
I think we are missing a few PRs here:
```suggestion
- [CDOT PR 13](https://github.com/CDOT-CV/jpo-geojsonconverter/pull/13): Update gjc to work with latest version of the jpo-ode 4.1.0
- [CDOT PR 12](https://github.com/CDOT-CV/jpo-geojsonconverter/pull/12): Adding Git history to CI Pipelines
- [CDOT PR 10](https://github.com/CDOT-CV/jpo-geojsonconverter/pull/10): Setting up CI with Azure Pipelines configuration.
- [USDOT PR 92](https://github.com/usdot-jpo-ode/jpo-geojsonconverter/pull/92): Processed PSM and Schema Generation
```

Reply:
Oh interesting. Good catch. I forgot those things did happen this quarter. Thanks!